frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -252,7 +252,7 @@ export const StyledSidebarContent = styled.div<StyledSidebarContentProps>(
     position: "relative",
     height: "100%",
     width: "100%",
-    overflow: hideScrollbar ? "hidden" : "overlay",
+    overflow: hideScrollbar ? "hidden" : "auto",
```

Comment:
I think your initial instinct in using the below makes sense here:
`overflow: hideScrollbar ? "hidden" : ["auto", "overlay"]`

It is also more consistent with how we handle the overflow property across streamlit.

Reply:
Oh okay sure thing makes sense to me!